src/lang/fr.json

```diff
@@ -308,5 +304,98 @@
   "FSC.Module.AboutTime.Configuration.Message": "Nous avons détecté que le module About-Time est/a été activé pour ce monde.<br/><br/>Si vous souhaitez importer le calendrier de About-Time vers Simple Calendar, choisissez \"Importer vers Simple Calendar\", cela fera correspondre Simple Calendar aux paramètres de About-Time.<br/><br/>Si vous souhaitez que About-Time corresponde à Simple Calendar, choisissez \" Exporter vers About-Time\", cela fera correspondre les paramètres de About-Time à la configuration de Simple Calendar.<br/>",
   "FSC.Module.AboutTime.Export": "Exporter vers About-Time",
   "FSC.Importer.NoFile": "Veuillez sélectionner un fichier à importer.",
-  "FSC.Importer.InvalidSCConfig": "Fichier non valide téléchargé, veuillez télécharger un fichier JSON exporté par Simple Calendar."
-}
+  "FSC.Importer.InvalidSCConfig": "Fichier non valide téléchargé, veuillez télécharger un fichier JSON exporté par Simple Calendar.",
```

Comment:
I would change this for "Le fichié téléversé est invalide, veuillez choisir un fichier JSON exporté par Simple Calendar"

Writing it this way is less redundant, and more accurate.